AEPCampaignClassic/Sources/Extensions/Data+256sha.swift

```diff
@@ -0,0 +1,40 @@
+/*
+ Copyright 2022 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+import CommonCrypto
+
+extension Data {
+
+    func sha256() -> String {
```

Comment:
+ 1 for the tests added them.
Also + 1 for consolidating to a single method (since multiple functions are not optimized running code)
But I like keeping both String and Data extensions for SHA hashing. bring more clarity and code separation.

Reply:
feels like this code belongs in `AEPServices`

Reply:
I was expecting SHA256 to be already there in AEPServices. But they use using FNV hashing.
For now, I will check with the core team to see if there is a release around and if so use this code from AEPServices for the 3.0.0 release.
If not, I will keep it here for now (will add a to-do comment). and we can make an update after the core is released with SHA hashing.

Let me know.